static/app/components/searchQueryBuilder/tokens/filter/filterOperator.tsx

```diff
@@ -89,17 +90,22 @@ function FilterKeyOperatorLabel({
   opLabel,
   includeKeyLabel,
 }: {
+  keyLabel: string;
   includeKeyLabel?: boolean;
-  keyLabel?: string;
   opLabel?: string;
 }) {
+  const {getFieldDefinition} = useSearchQueryBuilder();
+  const fieldDefinition = getFieldDefinition(keyLabel);
```

Comment:
I don't think this will work for `has` since the key label can be `does not have`. Maybe this needs an additional prop for for actual filter key instead of the label?